### DISCUSSION TITLE
Favorites cannot be shared

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#525] [Q&A] [answered]
WHEN searching for a favorite or pdi, you see it on the map but there is no option to share as before (coordinates, address, plus code, 3words, google maps, ...)

![Sense títol](https://github.com/devemux86/cruiser/assets/122691401/cca07765-0861-4199-9d13-2b0d7aec1fc8)
![Sense títol-1](https://github.com/devemux86/cruiser/assets/122691401/4e4e5244-bef4-420e-9ef5-3a074c5ddeb1)

**Comment by ramondevic** [comment]
Before there was...

![327970167-61306a07-9b05-4174-b7f9-cf077b490f85](https://github.com/devemux86/cruiser/assets/122691401/6ad3c6be-4645-4645-bbc8-a0a9ad6a103a)

**Comment by ramondevic** [comment]
Not on the favorites list either :(

**Reply by devemux86** [reply]
Favorites (or waypoints) list exports / shares files of favorites (or waypoints).

It never shared coordinates. If there is need for it, this will be a new feature.

**Reply by ramondevic** [reply]
But with versions 3.x it was possible to do it.

Why has this option been removed?

**Reply by devemux86** [reply]
@ramondevic 

Share location from the lists? No, it was never possible.
Unless you mean the select on map, which is different.

**Reply by ramondevic** [reply]
I'm not talking about sharing **locations**, I'm talking about sharing **favorites**

**Reply by devemux86** [reply]
With what app? What the other app receives?

The coordinates of the favorite? Or also something else?

**Reply by WolfgangZ42** [reply]
> I'm not talking about sharing locations, I'm talking about sharing favorites

It's possible - quite easy! 
I selected two of our hotels to share it with a friend:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/b899d9d8-0c9b-410b-8636-ce2b788512d9"/>
then typed on the three-dot-icon anywhere and got:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/082c3e4d-1df8-40cf-a343-a7c594a5cedd"/>
selected "... More" to see:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/60110fa3-0bd6-4659-9ec2-8eb88998409e"/>
selected "   Share":
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/9c4ed87e-00d7-4055-8d7c-528f08b29a17"/>
typed on "Test/Download" and sent it. 
That's it!

**Reply by devemux86** [reply]
@WolfgangZ42 this process shares items as files.

@ramondevic and @DolomitesTom probably mean to share item location as text.
Like the long press on map > Share process.

**Comment by devemux86** [comment]
@ramondevic please see here for how share any possible map location:
- https://github.com/devemux86/cruiser/discussions/522#discussioncomment-9320726

---

This was discussed (and tested in the last Beta) with all users, looking for ways to simplify the menus:
- https://github.com/devemux86/cruiser/discussions/496

**Comment by devemux86** [comment, marked as answer]
The show / share menu in the case of favorites and waypoints can be restored, as they have available space.

**Reply by ramondevic** [reply]
I'm sure that many users will be very happy to have the "Show / Share" option back when we are on a Favorite, with this you win easily and quickly.

Thanks !

![Screenshot_20240507-075134](https://github.com/devemux86/cruiser/assets/122691401/33c67166-7498-4fee-899c-11dd59b99b47)![Screenshot_20240507-074819](https://github.com/devemux86/cruiser/assets/122691401/eac7bc6f-26a6-49f7-82f6-6f80096d3ae7)
